src/app/pages/assets/asset/asset.component.ts

```diff
@@ -191,13 +181,21 @@ export class AssetComponent implements OnInit {
   }
 
   public loadAsset() {
+    // ID not provided we are in creation mode
     if (!this.currentAssetID) {
+      if (this.authorizationService.canCreateAsset()) {
```

Comment:
I think there is no need for this check (we are trying to get rid of the authorizationService on the frontend).

Reply:
If the plan is to remove the authorizationService then I agree, but in that case we might need to send a flag to the pop-up that the user is allowed to create an asset (when we retrieve the asset list)

Reply:
> If the plan is to remove the authorizationService then I agree, but in that case we might need to send a flag to the pop-up that the user is allowed to create an asset (when we retrieve the asset list)

I'm not sure if this going to be necessary if we display the create button only if the user is allowed to create (which is case)